troposphere/static/js/components/images/detail/header/HeaderView.react.js

```diff
@@ -1,14 +1,15 @@
 
 define(
   [
-    'react',
+    'react/addons',
     'backbone',
     'jquery',
     'stores',
+    'modals',
     // plugins
-    'bootstrap'
+    // 'bootstrap' // Not used?
```

Comment:
I think the reason we have this is for an _implicit_ include of `'bootstrap'` (but heck, I am **not** 100% sure). I see it mostly included for styles - but also for the model mixin.